tube_tipper.scad

```diff
@@ -57,9 +57,9 @@ module tube_tipper(number_tubes = 5) {
       for(y = tube_list) {
         translate([-wall_thickness,
             (y - (number_tubes+1)/2) * (tube_diameter + tube_spacing),
-            0])
+            0.25 * tube_spacing])
           rotate([0, 71, 0])
-          cylinder(d = 0.5 * tube_diameter, h = 2 * wall_thickness);
+          cylinder(d = 0.5 * (tube_diameter + tube_z_stretch), h = 2 * wall_thickness);
```

Comment:
I don't think this works, it will not introduce the right kind of stretch if you add it to the diameter. Better to add a `resize([0, 0, tube_diameter + tube_z_stretch])` above the rotate instead.

Reply:
Made this change, in the next commit